src/main/java/net/anotheria/anoprise/dataspace/DataspaceServiceImpl.java

```diff
@@ -51,7 +51,7 @@ public DataspaceServiceImpl() {
 		try {
 			cache = Caches.createConfigurableHardwiredCache("ano-prise-dataspace-cache");
 		} catch (IllegalArgumentException e) {
-			log.warn("Can't find cache configuration for ano-prise-dataspace-cache, falling back to min cache.");
+			log.warn("Can't find cache configuration for ano-prise-dataspace-cache, falling back to min cache.", e);
```

Comment:
exception was ignored on purpose, the log message already says what happened, no need to further spam.